src/CMakeLists.txt

```diff
@@ -69,12 +72,6 @@ add_custom_target(tags.py ALL
   VERBATIM
   )
 
-find_path(ANTLR_RUNTIME_INCLUDE antlr4-runtime.h
-  HINTS /usr/include/antlr4-runtime
-        $ENV{ANTLR4_RUNTIME_INCLUDE}
-  REQUIRED)
-include_directories(${ANTLR_RUNTIME_INCLUDE})
-
```

Comment:
This is only needed when using the system library, so it has been moved up where that decision is made.